starter/script.js

```diff
@@ -1 +1,60 @@
+var person = prompt("Welcome to the tour of the hippest neighborhood in DC, Columbia Heights! We are going to take an adventure through the best parts of the neighborhood. What is your name?");
+var transportation = prompt("Nice to meet you " + person + ". Would you like to take the bus, or walk during your adventure today?"); //1st question
+//option 1 -will be bus?
+if (transportation == "bus") {
+  //2nd question - cash or smarttrip
+  var payment = prompt("Playing it safe i see. That's allright. Most city busses cost $1.75. Are you going to pay with your SmartTrip, or cash?");
+  if (payment == "SmartTrip") {
+    var buspay = prompt("Uh oh, looks like your SmartTrip is out of money. That won't do us any good. Let's add some money to your card. Do you want to add money at Metro or at a busstop?"); //3rd question
+    if (buspay == "Metro") {
+      alert("Uh oh! There was a mysterious bag left at the Columbia Heights metro station, so no one is allowed to enter or leave!!! NO bus riding for you today");
+    } else if (buspay == "busstop") {
+      alert("Trick question. There are no SmartTrip kiosks at busstops. Your adventure is OVER before it even began!");
+    } else {
+      alert("That is not a valid option! try again");
```

Comment:
how does the user try again? by reloading the page? can you think of a way to restart your adventure when it gets here?